[East Sussex] Fix menu overlapping intro, and highlighted item looking rubbish

stevenday opened the pull request:
Fixes https://github.com/mysociety/FixMyStreet-Commercial/issues/659 and a
related issue I noticed at the same time where the currently selected item in
the menu had different styling and so was smaller and out of line.

dracos commented:
The menu also overlaps on other pages, e.g. help page, alerts page, I think a more general solution is needed than just in `#front-main`, I'm afraid. However, thanks especially for including the comment in layout explaining what the CSS is doing there (note everyone – do this! All the time!)

stevenday commented:
@dracos - thanks for spotting that, I've added some extra css to account for those pages too (and removed an irregularity with the menu padding). Hopefully that's it dealt with everywhere.

dracos commented:
Looks good :)
